src/activities/quizzes/QuizEntity.js

```diff
@@ -48,6 +48,18 @@ export class QuizEntity extends Entity {
 		}
 	}
 
+	/**
+	 * @returns {bool} Whether or not the user can preview a quiz
+	*/
+
+	canPreviewQuiz() {
+		if (!this._entity) {
+			return false;
+		}
+
+		return this._entity.hasLinkByRel('preview');
```

Comment:
could we make this a constant in  that hypermedia-constants file `Rels.Quizzes.preview` or something.